platform-includes/feature-flags/_default.mdx

```diff
@@ -0,0 +1,8 @@
+Please read the note below to ensure that you also complete one additional step.
+
+<Alert level="warning" title="Next Steps">
```

Comment:
Hmm why don't we have the set up for feature flag changes and evaluations instructions embedded on this integrations page? 🤔 

I think there's some consolidating we could do